src/main/java/com/askimed/nf/test/lang/extensions/GlobalMethods.java

```diff
@@ -38,7 +45,18 @@ public static void assertAll(Closure... closures) throws Exception {
 		}
 
 		if (failed > 0) {
-			throw new Exception(Integer.toString(failed) + " of " + Integer.toString(closures.length) + " assertions failed");
+			throw new PowerAssertionError(Integer.toString(failed) + " of " + Integer.toString(closures.length) + " assertions failed");
+		}
+	}
+	
+	public static void assertContainsInAnyOrder(List<Object> list, List<Object> expected) throws Exception {
```

Comment:
in documentation `assertInAnyOrder` vs. `assertContainsInAnyOrder`.  I would prefer `assertInAnyOrder`. What do you think?

Reply:
Happy with either really. I do think `assertInAnyOrder` sounds better, but really it's binding for Hamcrest's method, so I'm inclined to be consistent with that. 

@byb121  what do you think?

Reply:
It seems to me that Hamcrest already gained popularity with developers judging by the repo's stars. Although I like the shorter form as well, I think the longer form could be easier to guess its function for many devs.

Reply:
Good arguments, let us go with the longer form to be consistent with Hamcrest.

Reply:
Awesome! Thanks. If everything is fine from your side, I will merge the pull request

Reply:
Hey @lukfor  yeah all good for me!